viam-cartographer/src/carto_facade/carto_facade_test.cc

```diff
@@ -59,28 +50,29 @@ void viam_carto_config_teardown(viam_carto_config vcc) {
 
 viam_carto_algo_config viam_carto_algo_config_setup() {
     struct viam_carto_algo_config ac;
-    ac.optimize_on_start = true;
-    ac.optimize_every_n_nodes = 1;
-    ac.num_range_data = 2;
-    ac.missing_data_ray_length = 3.1;
-    ac.max_range = 4.1;
-    ac.min_range = 0.1;
-    ac.max_submaps_to_keep = 6;
-    ac.fresh_submaps_count = 7;
-    ac.min_covered_area = 8.1;
-    ac.min_added_submaps_count = 9;
-    ac.occupied_space_weight = 10.1;
-    ac.translation_weight = 11.1;
-    ac.rotation_weight = 12.1;
+    ac.optimize_on_start = false;
+    ac.optimize_every_n_nodes = 3;
+    ac.num_range_data = 100;
+    ac.missing_data_ray_length = 25.0;
+    ac.max_range = 25.0;
+    ac.min_range = 0.2;
+    ac.max_submaps_to_keep = 3;
+    ac.fresh_submaps_count = 3;
+    ac.min_covered_area = 1.0;
+    ac.min_added_submaps_count = 1;
+    ac.occupied_space_weight = 20.0;
+    ac.translation_weight = 10.0;
+    ac.rotation_weight = 1.0;
     return ac;
 }
 
-BOOST_AUTO_TEST_SUITE(CartoFacadeCPPAPI)
+BOOST_AUTO_TEST_SUITE(CartoFacadeCPPAPI, *boost::unit_test::disabled());
```

Comment:
Changed this so that `make test` doesn't run this test (as another github workflow manually runs the carto facade tests).